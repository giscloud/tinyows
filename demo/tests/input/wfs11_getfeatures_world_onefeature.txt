service=WFS&version=1.1.0&request=GetFeature&typename=tows:world&maxfeatures=1
