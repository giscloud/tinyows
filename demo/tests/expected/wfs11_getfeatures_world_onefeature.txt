Content-Type: text/xml; subtype=gml/3.1.1

<?xml version='1.0' encoding='UTF-8'?>
<wfs:FeatureCollection
 xmlns:tows='http://www.tinyows.org/'
 xmlns:wfs='http://www.opengis.net/wfs'
 xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
 xmlns:gml='http://www.opengis.net/gml'
 xmlns:xsd='http://www.w3.org/2001/XMLSchema'
 xmlns:ogc='http://www.opengis.net/ogc'
 xmlns:xlink='http://www.w3.org/1999/xlink'
 xmlns:ows='http://www.opengis.net/ows'
 xsi:schemaLocation='http://www.tinyows.org/
    http://127.0.0.1/cgi-bin/tinyows?service=WFS&amp;version=1.1.0&amp;request=DescribeFeatureType&amp;Typename=tows:world   http://www.opengis.net/wfs
   http://schemas.opengis.net/wfs/1.1.0/wfs.xsd
   http://www.opengis.net/gml
   http://schemas.opengis.net/gml/3.1.1/base/gml.xsd'
>
  <gml:featureMember>
   <tows:world gml:id="world.1">
   <tows:name>Afghanistan</tows:name>
   <tows:gmi_cntry>AFG</tows:gmi_cntry>
   <tows:region>Asia</tows:region>
   <tows:geom><gml:MultiSurface srsName="urn:ogc:def:crs:EPSG::4326"><gml:surfaceMember><gml:Polygon><gml:exterior><gml:LinearRing><gml:posList>37.333199 65.627296 37.458881 65.646927 37.53693 65.70137 37.534161 65.766083 37.568878 65.785522 37.323601 66.30275 37.360512 66.538765 37.368038 66.586906 37.338322 66.665253 37.36137 66.744423 37.377201 67.021637 37.246651 67.200256 37.191929 67.229424 37.185261 67.266373 37.234989 67.424408 37.27248 67.521637 37.215542 67.557457 37.24609 67.649406 37.206089 67.774139 37.185799 67.777153 37.088322 67.791641 37.061371 67.885529 37.014431 67.910522 36.924709 68.035522 37.019981 68.28331 37.086929 68.278313 37.111099 68.301643 37.104431 68.413017 37.148041 68.412193 37.16193 68.524979 37.246929 68.654694 37.247761 68.825813 37.322491 68.809402 37.324711 68.85025 37.272209 68.904968 37.28471 68.924408 37.31749 68.891647 37.307758 68.999687 37.094151 69.246628 37.124981 69.333038 37.22998 69.45137 37.344151 69.381348 37.455818 69.382736 37.580818 69.515808 37.611931 69.91803 37.564991 69.958588 37.529148 70.134979 37.62109 70.255539 37.741371 70.276917 37.872211 70.165527 37.94165 70.170242 37.92054 70.203857 38.057758 70.378304 38.16721 70.491913 38.278591 70.584412 38.334148 70.596359 38.375259 70.685516 38.414429 70.672203 38.453041 70.880524 38.40609 71.101913 38.248871 71.363022 38.18803 71.363861 38.019981 71.284698 37.918049 71.264137 37.905819 71.37468 37.942478 71.546356 37.903042 71.591919 37.81609 71.588577 37.764431 71.528847 37.284161 71.49025 37.20776 71.446907 37.05859 71.43219 36.741371 71.568314 36.672211 71.695236 36.692478 71.842743 36.812199 72.011368 36.989979 72.346626 37.026089 72.663589 37.22998 72.798859 37.320541 73.08136 37.400822 73.150528 37.463322 73.314407 37.434429 73.776642 37.331379 73.752762 37.304428 73.649406 37.26276 73.619957 37.239979 73.634407 37.22942 73.792473 37.4286 74.363861 37.399429 74.400803 37.39193 74.699692 37.240299 74.898827 37.219151 74.816933 37.30109 74.763313 37.266102 74.669418 37.234711 74.651077 37.243038 74.537193 37.175259 74.392197 37.139149 74.400528 37.02647 74.567749 36.96526 74.558006 37.011101 74.482193 36.958881 74.336906 36.899429 74.251923 36.906651 74.158859 36.821659 74.06218 36.837761 73.963028 36.901089 73.779694 36.873871 73.113022 36.771931 72.492462 36.71471 72.181641 36.6572 72.191643 36.628868 72.073303 36.58942 72.074692 36.491928 71.795807 36.41666 71.817749 36.40749 71.753036 36.46804 71.646637 36.372478 71.563019 36.335819 71.582748 36.133041 71.248581 36.047211 71.188019 35.96859 71.291077 35.946091 71.380524 35.711102 71.545242 35.626652 71.508591 35.561932 71.613861 35.48193 71.606918 35.436939 71.647202 35.32832 71.549133 35.28915 71.553574 35.207489 71.659126 35.131371 71.618027 35.0947 71.543297 34.959431 71.496078 34.886929 71.313858 34.744431 71.225517 34.676651 71.09581 34.56805 71.094971 34.55859 70.995796 34.510818 70.978027 34.39415 71.073029 34.36137 71.153313 34.166088 71.135529 34.008881 70.981079 34.013321 70.905533 33.973591 70.900528 33.943039 70.491348 34.031101 69.902763 33.881931 69.907471 33.75304 69.985519 33.735538 70.132736 33.64082 70.196617 33.485821 70.197739 33.396099 70.306084 33.33194 70.32692 33.202492 70.144974 33.20499 70.067749 33.139431 70.033577 33.089981 69.881073 33.126942 69.791351 33.081928 69.561081 33.008598 69.492462 32.873878 69.515259 32.773869 69.394974 32.682209 69.456917 32.587761 69.397751 32.526379 69.287483 32.443871 69.248016 32.21777 69.283043 32.069149 69.287483 31.94389 69.333862 31.64555 69.029694 31.60388 68.833038 31.69944 68.731918 31.778879 68.712479 31.829161 68.54747 31.79472 68.444122 31.77277 68.447739 31.76527 68.57164 31.72666 68.53775 31.81559 68.221626 31.83305 68.166077 31.63583 67.98053 31.63999 67.887482 31.53083 67.738022 31.5336 67.581093 31.40999 67.646362 31.4111 67.766922 31.38249 67.799698 31.341101 67.791924 31.2136 67.288017 31.22249 67.256363 31.239161 67.069702 31.25444 67.034698 31.297779 67.051643 31.318609 67.036911 31.296101 66.891647 31.21221 66.723022 31.086109 66.684128 30.97777 66.56636 30.94611 66.404968 30.57527 66.281921 30.45055 66.35025 30.114161 66.259697 30.071381 66.238297 29.966101 66.361359 29.85194 66.256638 29.640551 65.417473 29.5411 65.034126 29.58638 64.695511 29.52389 64.273857 29.48749 64.193024 29.394159 64.131348 29.50388 63.5872 29.43388 62.78693 29.406099 62.484428 29.615829 61.743599 29.826099 60.99387 29.862431 60.866871 30.259991 61.25304 30.843611 61.812759 30.94582 61.803871 31.04888 61.84721 31.24527 61.76582 31.318331 61.771099 31.383329 61.7136 31.395821 61.619431 31.498329 60.843868 31.8736 60.810261 32.234718 60.857769 32.578869 60.74165 33.071659 60.581928 33.16304 60.591091 33.418049 60.85165 33.49387 60.858318 33.504429 60.9286 33.554428 60.901649 33.574989 60.649151 33.653042 60.522758 33.739151 60.50526 33.81332 60.55415 33.999149 60.52137 34.150822 60.513599 34.313599 60.67165 34.31638 60.911091 34.345821 60.89777 34.483318 60.756649 34.54166 60.733601 34.576389 60.863602 34.812759 61.065262 35.201649 61.11388 35.279148 61.104431 35.29694 61.1861 35.481091 61.239151 35.52026 61.27776 35.60675 61.278721 35.639709 61.365822 35.52721 61.464989 35.450821 61.576649 35.411091 61.802479 35.45998 61.97221 35.394711 62.100819 35.297771 62.259159 35.145538 62.305538 35.286381 62.459431 35.224979 62.62331 35.481369 63.110538 35.626091 63.095261 35.69553 63.23888 35.825821 63.10416 35.861931 63.119431 35.852211 63.317211 35.90971 63.538879 35.9622 63.595539 36.03915 63.933868 36.000271 64.063858 36.088039 64.059692 36.167488 64.169418 36.15192 64.282471 36.216381 64.32164 36.247211 64.458862 36.35638 64.570534 36.45998 64.627747 36.629429 64.61525 36.923038 64.795242 37.095821 64.779137 37.124981 64.798019 37.244431 65.072746 37.2486 65.530807 37.333199 65.627296</gml:posList></gml:LinearRing></gml:exterior></gml:Polygon></gml:surfaceMember></gml:MultiSurface></tows:geom>
   </tows:world>
  </gml:featureMember>
</wfs:FeatureCollection>
